packages/nx-plugin/src/generators/configuration/__snapshots__/code-pushup-config.integration.test.ts.snap

```diff
@@ -0,0 +1,20 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`generateCodePushupConfig options > should create code-pushup.config.ts with options in tree 1`] = `
+"import type { CoreConfig } from '../dist/packages/models;
+import * as myPlugin from my-plugin";
+import {myPluginCategory} from "my-plugin";
+
+// see: https://github.com/code-pushup/cli/blob/main/packages/models/docs/models-reference.md#coreconfig
+export default {
+    persist: {
+  filename: "report-123"
+},
+    update: {
+  apiKey: "123"
+},
+    plugins: [myPlugin({ timeout: 42})],
+    categories: [myPluginCategory()]
+} satisfies CoreConfig;
+"
+`;
```

Comment:
Would be easier to read a formatted `.ts` file as the snapshot (same for unit test snapshot) :pray:

Reply:
the integration are updated

Reply:
I meant `code-pushup-config.integration.test.ts.snap` and `code-pushup-config.unit.test.ts.snap`. But I see you've delete both in caaacc2c973d19a28df0ebab276f8ae7c49c5b00, so I guess they were outdated anyway.

Reply:
yes! tests got refactored and files splitted